src/v/config/config_store.h

```diff
@@ -136,6 +136,36 @@ class config_store {
         w.EndObject();
     }
 
+    void to_json_for_metrics(json::Writer<json::StringBuffer>& w) {
+        w.StartObject();
+
+        for (const auto& [name, property] : _properties) {
+            if (property->get_visibility() == visibility::deprecated) {
+                continue;
+            }
+
+            if (property->type_name() == "boolean") {
+                w.Key(name.data(), name.size());
+                property->to_json(w, redact_secrets::yes);
+                continue;
+            }
+
+            if (property->is_nullable()) {
+                w.Key(name.data(), name.size());
+                w.String(property->is_default() ? "nullopt" : "[value]");
```

Comment:
Not all `std::optional` properties default to nullopt

Reply:
Fix it to "default", "[value]"
Will chat about 
```
{
   'is_default': true/false  # Always set this
   'value': '' # Only set this for bools and enums
}
```

Today later